packages/nuxt/src/module.ts

```diff
@@ -66,7 +66,7 @@ export default defineNuxtModule<ModuleOptions>({
       if (moduleOptions.debug) {
         // eslint-disable-next-line no-console
         console.log(
-          `[Sentry] Using your ${serverConfigFile} file for the server-side Sentry configuration. In case you have a \`public/instrument.server\` file, the \`public/instrument.server\` file will be ignored. Make sure the file path in your node \`--import\` option matches your Sentry server config file.`,
+          `[Sentry] Using your \`${serverConfigFile}\` file for the server-side Sentry configuration. In case you have a \`public/instrument.server\` file, the \`public/instrument.server\` file will be ignored. Make sure the file path in your node \`--import\` option matches the Sentry server config file in your \`.output\` folder and has a \`.mjs\` extension.`,
```

Comment:
Maybe we can simplify this by just telling people what their new `--import` should look like?

Reply:
That's hard to know at this point and I would add this to the docs when we know more about this. Locally, it has to be added as `server/sentry.server.config.mjs` but on deployment targets like netlify it could be located somewhere else.

Reply:
Ah, right. Guess we can do this later then.